emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+from builtins import object
+import urllib.request, urllib.parse, urllib.error, urllib.request, urllib.error, urllib.parse
+import logging
+import json
+import emission.core.get_database as edb
+import emission.core.common as cm
+import unittest
+
+from emission.core.wrapper.trip_old import Coordinate
+
+import emission.net.ext_service.geocoder.nominatim as eco
+
+class NominatimTest(unittest.TestCase):
+
+    def test_make_url_geo(self):
+        expected_result = "http://nominatim.openstreetmap.org/search?q=Golden%2C+Colorado&format=json"
```

Comment:
I don't think we should hardcode `nominatim.openstreetmap.org` for all these tests.
It is OK to hardcode them for a few tests (to test the fallback if the env variable is not present), but we should really be setting the appropriate environment variable and expect the value accordingly.

Reply:
Went through and replaced with enviornment variable, which will be set to localhost URL during testing. Also currently going through and replacing all of the tests with locations in rhode island, so that they can call through the docker run server on GH.

Reply:
You mean that you are changing all the real_data tests to rhode island?

That is not a good idea:
1. it is not real data any more
2. the location end points will no longer match up with the trajectories, unless
    3. you also move over the trajectories, which is an incredible amount of work

You should use fake trips instead. I have provided you with examples of tests that create fake trips.

Reply:
No, I am not changing any of the real_data tests. I realize now how vague my comment was. In this file, I'm changing the unit tests to query locations/place names/latlon in rhode island, rather than those in California or Colorado. None of the Nominatim unit tests use a ground truth file.